README.md

```diff
@@ -1,66 +1,22 @@
-README
-======
+# ownCloud Calendar 
 
-[![Build Status](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/build.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar-rework/build-status/master)[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar-rework/?branch=master)[![Code Coverage](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar-rework/?branch=master)
+[![Build Status](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/build.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar-rework/build-status/master)
```

Comment:
please change from calendar-rework to calendar
(here as well as the following two lines)